src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -108,6 +108,8 @@ export interface ICloseAllEditorsOptions {
 export interface IEditorReplacement {
 	editor: IEditorInput;
 	replacement: IEditorInput;
+	// Skips asking the user for confirmation and doesn't save the document. Only use this if you really need to!
+	skipSaveDialog?: boolean;
```

Comment:
I find `skipSaveDialog` not very intuitive because the dialog also allows to revert. I would rename this to `forceReplaceDirty` which:
* makes it clear this is an option that needs to be used with care because it forces certain behaviour that is not standard
* talks about "dirty" editors which is what this is about

Reply:
Renamed